sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -230,6 +230,16 @@ class InMemoryCatalogedDDLSuite extends DDLSuite with SharedSparkSession {
       }
     }
   }
+
+  test("SPARK-40719: CTAS should respect TBLPROPERTIES during query execution") {
+    withTable("t") {
+      withTempDir { dir =>
+        sql(s"CREATE TABLE t USING PARQUET LOCATION '$dir' " +
+          "TBLPROPERTIES (parquet.compression 'zstd') AS SELECT 1")
```

Comment:
I agree that it's better to be consistent between the Spark native CREATE TABLE syntax and the Hive one, but we probably should revisit the Hive syntax first. If it's reasonable (treat TBLPROPERTIES as write options), we can follow it in the native syntax